html/editing/editing-0/contenteditable/contentEditable-queryCommandEnabled.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Editing: contentEditable queryCommandEnabled test</title>
+    <link
+            rel="help"
+            href="https://html.spec.whatwg.org/multipage/#contenteditable"
+    />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <div id="contentEditableTrue" contenteditable="true">true</div>
+    <div id="contentEditablePlaintextOnly" contenteditable="plaintext-only">plaintext-only</div>
+</head>
+<body>
+<script>
+    var selection = document.getSelection();
+    var contentEditableTrue = document.getElementById('contentEditableTrue');
+    var contentEditablePlaintextOnly = document.getElementById('contentEditablePlaintextOnly');
+
+    function checkQueryCommandEnabled(el, title, command, expectQueryCommandEnabled) {
+        test(() => {
+            selection.selectAllChildren(el);
+            assert_equals(document.queryCommandEnabled(command), expectQueryCommandEnabled);
+        }, title);
+    }
+
+    checkQueryCommandEnabled(contentEditableTrue, "document.queryCommandEnabled('bold') is true for contenteditable=true", 'bold', true);
+    checkQueryCommandEnabled(contentEditablePlaintextOnly, "document.queryCommandEnabled('bold') is false for contenteditable=plaintext-only", 'bold', false);
```

Comment:
I think repetition can be avoided here, e.g. with two `test()`s in `checkQueryCommandEnabled`